src/steps/analyze-project/analyze-components/find-arguments.ts

```diff
@@ -14,13 +31,13 @@ export function findArguments({
     return;
   }
 
-  // ...
+  const args = new Set<string>();
 
-  if (classFile === undefined) {
-    return;
-  }
```

Comment:
**bugfix**

Shouldn't have made an early exit when `classFile` is `undefined`.